dev/com.ibm.ws.jaxws.2.3.clientcontainer/bnd.bnd

```diff
@@ -9,29 +9,29 @@
 #     IBM Corporation - initial API and implementation
 #*******************************************************************************
 -include= ~../cnf/resources/bnd/bundle.props
-          
+exportVer=2.3
 bVersion=1.0
 
 Bundle-Name: JAX-WS ClientContainer
 Bundle-SymbolicName: com.ibm.ws.jaxws.2.3.clientcontainer
 Bundle-Description: IBM JAX-WS ClientContainer; version=${bVersion}
-
-
+Bundle-Activator: org.apache.cxf.bus.osgi.CXFActivator
 
 # in cxf-rt-frontend-jaxws-2.6.2 bundle, we exposed: 
 # META-INF/services/javax.xml.ws.spi.Provider = com.ibm.ws.jaxws.client.LibertyProviderImpl
 # so we need make com.ibm.ws.jaxws.client;thread-context=true
 Export-Package: \
-    com.ibm.ws.jaxws, \
-    com.ibm.ws.jaxws.bus, \
-    com.ibm.ws.jaxws.client;thread-context=true, \
-    com.ibm.ws.jaxws.client.injection, \
-    com.ibm.ws.jaxws.metadata, \
-    com.ibm.ws.jaxws.metadata.builder, \
-    com.ibm.ws.jaxws.security, \
-    com.ibm.ws.jaxws.support, \
-    com.ibm.ws.jaxws.utils, \
-    com.ibm.wsspi.jaxws
+    com.ibm.ws.jaxws;version=${exportVer},\
```

Comment:
Ideally, jaxws.2.3.clientcontainer would remove the identical classes that are in jaxws.clientcontainer and only have the ones that are different like what we did with some other components.

Reply:
Totally right, and I mention this in a later comment, but I would like to go further in that there is common code shared across both versions and across the server/client containers.

Reply:
For these `com.ibm.ws.jaxws*`, could you use `package-info.java` files (with the OSGI versions in them) instead of exporting them here?